### PR TITLE
[task/37870] - update What is Quix topic 

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Update the "What is Quix?" topic.

* [Ticket](https://app.shortcut.com/quix/story/37870/update-what-is-quix-page)

## Review

Review the page for correct product naming. Is there more we want to add? 

* [Page to review](https://quixdocsdev.blob.core.windows.net/pr127/platform/what-is-quix.html)

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ nav:
   - Intro: 'index.md'
   - 'Quickstart': 'platform/tutorials/quick-start/quick-start.md'
   - Platform:
-    - 'What is Quix?': 'platform/intro.md'
```
Comment (SteveRosam) [inline mkdocs.yml:18]: you should probably add a redirect

File: docs/platform/what-is-quix.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# What is Quix Platform?
+
+Quix Platform is a complete system for developing and deploying applications for the processing of real-time streaming data. Quix Platform includes an online IDE for developing, debugging, and deploying your real-time streaming applications. In addition, Quix also provides an open source library called Quix Streams.Quix Streams is the client library that you use in your Python or C# code to develop custom elements of your processing pipeline.
```
Comment (SteveRosam) [inline docs/platform/what-is-quix.md]: Needs a space here -> Quix Streams.Quix Stream

Comment (SteveRosam) [reply docs/platform/what-is-quix.md]: could link to QuixStreams on first mention

File: docs/platform/what-is-quix.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+Quix Platform was built on top of a message broker, specifically [Kafka](../client-library/kafka.md), rather than on top of a database, as databases introduce latency that can result in problems in real-time applications, and can also present scaling issues. Scaling databases for real-time applications also introduces more cost and complexity. Quix Platform helps abstract these issues, providing you with a scaleable and cost effective solution. 
+
+Broker technology allows the developer to work with live data in memory, rather than data retrieved using complex queries from disk. This is much faster and therefore more suited to real-time applications. However, the problem with broker technologies can be that they are more complex to use. Quix Platform and Quix Streams provide abstractions and tools that make working with your real-time data processing much easier than working with broker technology directly.
+
+Quix also treats Python developers as first-class citizens, making it easier for Python developers to work with real-time data using the abstractions and tools they are already familiar with, such as using the pandas dataframe format.
```
Comment (SteveRosam) [inline docs/platform/what-is-quix.md]: Pandas? (uppsercase P)

File: docs/platform/what-is-quix.md (hunk 2 above)
Comment (tbedford) [reply docs/platform/what-is-quix.md]: Well spotted. Fixed. Thanks.

File: mkdocs.yml (hunk 1 above)
Comment (tbedford) [reply mkdocs.yml:18]: Done!

**Review by SteveRosam — APPROVED** [review]

File: docs/platform/what-is-quix.md (hunk 3 above)
Comment (tbedford) [reply docs/platform/what-is-quix.md]: It's lower case on their site, so I went with that.

**Pull request merged**